### PR TITLE
upgrading env in 2022

### PR DESCRIPTION
**Pull request by wkerzendorf**
stuck at 
```python
/Users/wkerzend/python/tardis/tardis/io/atom_data/base.py(525)prepare_atom_data()
-> self.yg_data = self.yg_data.loc[self.selected_atomic_numbers]
```

### CODE REVIEW

File: tardis/visualization/tools/tests/test_sdec_plot.py [hunk 1]
```diff
@@ -408,12 +408,16 @@ def test_generate_plot_mpl(
         species_list : list of str
         """
         subgroup_name = make_valid_name("mpl" + request.node.callspec.id)
+        if distance is not None:
+            observed_spectrum = observed_spectrum
+        else:
+            observed_spectrum = None
```
Comment (atharva-2001) [inline tardis/visualization/tools/tests/test_sdec_plot.py:414]: Why not
```suggestion
        if distance is None:
            observed_spectrum = None
```

File: tardis/io/tests/test_HDFWriter.py [hunk 2]
```diff
@@ -79,7 +77,8 @@ def test_multi_index_write(tmpdir):
     actual.to_hdf(fname, path="test", overwrite=True)
     expected = pd.read_hdf(fname, key="/test/mock_hdf/property")
     expected = pd.MultiIndex.from_tuples(expected.unstack().values)
-    pdt.assert_almost_equal(actual.property, expected)
+    # These are multiindex objects, so we need to compare the values
+    np.all(expected.values == actual.property.values)
```
Comment (jvshields) [inline tardis/io/tests/test_HDFWriter.py:81]: Does this actually test anything? I think this test is missing an assert.

Comment (wkerzendorf) [reply tardis/io/tests/test_HDFWriter.py:81]: correct.

**Review by atharva-2001 — APPROVED** [review]
Thank you for the pull request!

**Review by andrewfullard — APPROVED** [review]

**Review by jvshields — APPROVED** [review]

**Pull request merged**